executers/gopackage.go

```diff
@@ -101,3 +105,35 @@ func (dependencyPackage *Package) Publish(summary string, targetRepo string, ser
 func (dependencyPackage *Package) Dependencies() []buildinfo.Dependency {
 	return dependencyPackage.buildInfoDependencies
 }
+
+// Adds the mod, zip and info files as build info dependencies
+func (dependencyPackage *Package) CreateBuildInfoDependencies(includeInfoFiles bool) error {
+	// Mod file dependency for the build-info
+	modDependency := buildinfo.Dependency{Id: dependencyPackage.id}
+	checksums, err := checksum.Calc(bytes.NewBuffer(dependencyPackage.modContent))
```

Comment:
In the above line, wer're using *checksum.Calc* to calculate the checksum of the info file.
Later in this function, we're doing the same for the mod and zip files, but we're doing it differently, using the *fileutils.GetFileDetails* API.
What is the reason for this change?

Reply:
The mod is calculated with checksum since the mod is not a file but a content:
`checksums, err := checksum.Calc(bytes.NewBuffer(dependencyPackage.modContent))`
The other two are path to files and therefore being calculated with fileutils.GetFileDetails.